### PR TITLE
Cleanup theme variables, archetypes, indentation

### PR DESCRIPTION
**Pull request by etiennetremel**
- move archetypes to the fikaworks template
- rename list.html.html to list.html and adjust indentation
- added min hugo version to the theme
- cleanup theme variables

### CODE REVIEW

File: themes/fikaworks/archetypes/members/index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+name: "{{ .Name }}"
+thumbnail: "images/profile.jpg"
+specialties: ""
+member_status: "member"
+linkedin: ""
```
Comment (jasonrichardsmith) [inline themes/fikaworks/archetypes/members/index.md:6]: I think the "if" logic is not setup for empty variables just them not being set.  I could be wrong.

File: themes/fikaworks/layouts/members/list.html.html [hunk 2]
```diff
@@ -1,86 +0,0 @@
-{{ define "main" }}
```
Comment (jasonrichardsmith) [inline themes/fikaworks/layouts/members/list.html.html:1]: we could have just done a git mv "stop erasing my history!" :sob:

**Review by jasonrichardsmith — APPROVED** [review]

**Review by jocatalin — APPROVED** [review]

File: themes/fikaworks/archetypes/members/index.md (hunk 1 above)
Comment (etiennetremel) [reply themes/fikaworks/archetypes/members/index.md:6]: changed the logic to "with" as suggested [here](https://discourse.gohugo.io/t/solved-isset-how-to-check-variable-is-empty/1764/6)

File: themes/fikaworks/layouts/members/list.html.html (hunk 2 above)
Comment (etiennetremel) [reply themes/fikaworks/layouts/members/list.html.html:1]: that's ok, reverted and used git mv, I will probably add prettier one in the coming days to keep the indentation consistent.

**Review by jasonrichardsmith — APPROVED** [review]

**Pull request merged**